test/unit/Ed25519PrivateKey.js

```diff
@@ -15,6 +15,8 @@ const DER_PRIVATE_KEY_BYTES = new Uint8Array([
 const DER_PUBLIC_KEY =
     "302a300506032b65700321004a6892f034d2d1c9b1a76acca8e34884055172f4210a0c02e3c7d55084f224d1";
 
+const STRESS_TEST_ITERATION_COUNT = 100;
```

Comment:
Try a bigger value here. There are no heavy calls. In theory you could even try more than 10^7.

Reply:
It takes a lot of time :( 10 000 tests take 50 seconds to run. Therefore 1000 test take like 5 seconds. 

Now that I think about it... considering the CI takes 30 minutes to run, then 1 minute per test case isn't that much. 

But they are 8 tests in total, so basically 8 minutes addition.

Reply:
100 is too small and doesn't guarantee that the method works correctly.
Increase tests to 1000 and timeout to 200000.